lambdas/data-migration2/tests/test-pdrs.js

```diff
@@ -143,7 +151,7 @@ test.serial('migratePdrRecord correctly migrates PDR record', async (t) => {
   const testPdr = generateTestPdr({
     collectionId: buildCollectionId(testCollection.name, testCollection.version),
     provider: testProvider.name,
-    execution: execution.arn,
+    execution: executionURL,
```

Comment:
So it was passing before because we were creating the dynamo record's `execution` field using the `arn` from the created PG record. Thus the lookup for an `arn` using the dynamo `execution` worked, even though in reality it never would. Just making sure I understand.

Reply:
Right. We took the `arn` that `fakeExecutionRecordFactory` produced and basically searched for executions by `pgExecution.arn === pgExecution.arn`, which works fine in test.  

I think the misunderstanding was that a PDR's `execution` field is NOT an ARN. It's a full URL. So comparing that to the Execution's `arn` field is a problem.